dynaswap-omrs/omod/src/main/java/org/openmrs/module/dynaswap/web/controller/TestController.java

```diff
@@ -0,0 +1,49 @@
+package org.openmrs.module.dynaswap.web.controller;
+
+import java.io.IOException;
+// import java.net.http.HttpHeaders;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.codehaus.jackson.JsonGenerationException;
+import org.codehaus.jackson.JsonProcessingException;
+import org.codehaus.jackson.map.JsonMappingException;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.openmrs.module.dynaswap.atallah.CryptDAG;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller("${rootrootArtifactId}.TestController")
```

Comment:
are you sure this works, because it seems like rootrootArtifactId is being repeated. Why not simply use the org.openmrs.module.dynaswap here?